File: Java/app/src/main/java/visprolabweek1/Fish.java
package main.java.visprolabweek1;
// Konsep: Inheritance tapi sudah masuk ke Polymorphism
// Child class mewarisi atribut & method dari parent
public class Fish extends Animal{
    @Override
    public void makeSound() {
        System.out.println("Blub");
    }
    
}

File: Dart/dart_application_1/bin/fish.dart
// Konsep: Inheritance
//
// Penjelasan:
// - Dog mewarisi dari Animal → punya semua method & properti Animal.
// - Harus implementasi makeSound() karena abstract.

import 'animal.dart';

class Fish extends Animal {
  @override
  void makeSound() {
    print('Blub');
  }
}